### PR TITLE
Add --command argument for the debugger

### PR DESCRIPTION
**Pull request by gergocs**
--command takes one string which contains commands to be executed at the start of the program.
The commands must be separated with semicolon.

Signed-off-by: Gergo Csizi gergocs@inf.u-szeged.hu

### CODE REVIEW

**Review by ksh8281 — APPROVED** [review]

File: tools/debugger/debugger_tester.sh [hunk 1]
```diff
@@ -8,7 +8,11 @@ RESULT_TEMP=`mktemp ${TEST_CASE}.out.XXXXXXXXXX`
 if [[ $IS_CLIENT_SOURCE == 0  ]] && [[ $TEST_CASE != *"tools/debugger/tests/client_source_multiple"* ]]; then
   ${ESCARGOT} --start-debug-server ${TEST_CASE}.js &
   sleep 1s
-  (cat "${TEST_CASE}.cmd" | ${DEBUGGER_CLIENT} --non-interactive) >${RESULT_TEMP} 2>&1
+  if [[ $TEST_CASE == *"tools/debugger/tests/do_command"* ]]; then
+    (echo -n -e "\x0a" | ${DEBUGGER_CLIENT} --non-interactive --command "e i;b do_command.js:2;c;c;c") >${RESULT_TEMP} 2>&1
+  else
```
Comment (clover2123) [inline tools/debugger/debugger_tester.sh:13]: It seems that this `command` test actually does not run in CI.
Please check the test runner (`tools/run-tests.py`)

File: tools/debugger/tests/do_command.expected [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Connecting to: localhost:6501
+Connection created!!!
+Stopped at tools/debugger/tests/do_command.js:2
+Breakpoint 1 at tools/debugger/tests/do_command.js:2
+0Print: 0
+Stopped at breakpoint:1 tools/debugger/tests/do_command.js:2
```
Comment (clover2123) [inline tools/debugger/tests/do_command.expected:6]: Some initial part of expected result seems incorrect.
According to your predefined command argument, `e i;b do_command.js:2;c;c;c`,
result order should be shown as follow
1) result of evaluation `i`
2) add break point at line 2
3) continue the execution ...

But the actual result is,
1) add break point as line 2
2) result of evaluation `i`
3) continue the execution ...

Comment (clover2123) [reply tools/debugger/tests/do_command.expected:6]: (One more thing)
At the start of debugger, it first stoped at `tools/debugger/tests/do_command.js:2`
But IMO it should be stop at line 1 not line 2.

**Review by zherczeg — COMMENTED** [review]
LGTM (informally)

**Review by clover2123 — APPROVED** [review]

**Pull request merged**